tests/e2e/cypress/integration/nested/example2.test.js

```diff
@@ -1,10 +0,0 @@
-context('Actions 2', () => {
```

Comment:
Is there a reason why this file is deleted ?

Reply:
I delete this file unintentionally. I returned this file back.